queue_services/strr-pay/devops/gcp/clouddeploy.yaml

```diff
@@ -0,0 +1,75 @@
+# Copyright 2022 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+apiVersion: deploy.cloud.google.com/v1
+kind: DeliveryPipeline
+metadata:
+ name: strr-pay-pipeline
+description: Deployment pipeline
+serialPipeline:
+ stages:
+ - targetId: bcrbk9-dev
+   profiles: [dev]
+   strategy:
+    standard:
+      verify: false
+   deployParameters:
+   - values:
+      deploy-env: "development"
+      deploy-project-id: "bcrbk9-dev"
+      service-name: "strr-pay-dev"
+      container-name: "strr-pay-dev"
+      cloudsql-instances: "bcrbk9-dev:northamerica-northeast1:strr-db-dev"
+      service-account: "sa-api@bcrbk9-dev.iam.gserviceaccount.com"
+ - targetId: bcrbk9-test
+   profiles: [test]
+   strategy:
+    standard:
+      verify: false
+   deployParameters:
+   - values:
+      deploy-env: "test"
+      deploy-project-id: "bcrbk9-test"
+      service-name: "strr-pay-test"
+      container-name: "strr-pay-test"
+      cloudsql-instances: "bcrbk9-test:northamerica-northeast1:strr-db-test"
+      service-account: "sa-api@bcrbk9-test.iam.gserviceaccount.com"
+ - targetId: bcrbk9-sandbox
+   profiles: [sandbox]
+   strategy:
+    standard:
+      verify: false
+   deployParameters:
+   - values:
+      deploy-env: "sandbox"
```

Comment:
i think, your deploy-env values need to correspond to values in config: https://github.com/bcgov/STRR/blob/67475ec9927eec56607ba8b9d567f5b94470b55b/queue_services/strr-pay/src/strr_pay/config.py#L58 so, you have production (or staging), testing, development there, but no 'sandbox' and 'test'. So probably, 'sandbox' need to be replaced with 'staging'